pyterrier/apply_base.py

```diff
@@ -376,14 +376,14 @@ def transform_iter(self, inp: pt.model.IterDict) -> pt.model.IterDict:
         for row in inp:
             row = row.copy()
             if "query" in row:
-                row = pt.model.push_queries_dict(row, inplace=True, keep_original=True)
+                row = pt.model.push_queries_dict(row, keep_original=True)
```

Comment:
This should be inplace to avoid making an extra copy, right?

Reply:
i disabled a lot of inplace=True, to avoid the temptation for downstream users. 

however, there are places where it used to make sense, e.g. if a row was new anyway. thoughts?

Reply:
I don't feel particularly strongly